### ISSUE TITLE
focal loss implement

### ISSUE THREAD

**Issue by DonghoonPark12** [pierluigiferrari/ssd_keras#273]
I use below code for focal loss.
```python
    def focal_loss(self, y_true, y_pred, gamma=2.0, alpha=0.75):
        # y_pred (tensor of shape (batch size, num_boxes, num_classes)
        eps = K.epsilon()
        y_pred = K.clip(y_pred, eps, 1. - eps)
        pt = tf.where(tf.equal(y_true, 1), y_pred, 1 - y_pred)
        loss = -K.pow(1. - pt, gamma) * K.log(pt)
        loss = alpha * loss
        return tf.reduce_sum(loss, axis=-1)
```
but compared to original mAP(I got 79.7mAP), I only got 0.787mAP. 

Any Idea that I can get Improved result??
